integration-tests/common/common.go

```diff
@@ -153,7 +153,7 @@ func (c *Common) CreateJobsForContract(cc *ChainlinkClient, observationSource st
 
 	oracleSpec := job.OCR2OracleSpec{
 		ContractID:                  ocrControllerAddress,
-		Relay:                       relay.Starknet,
+		Relay:                       relay.StarkNet,
```

Comment:
Yeah I replaced this where I could but I had to revert a couple places since they require changes in `core` first: https://github.com/smartcontractkit/chainlink-starknet/pull/222/commits/46e19feda3eca9d78a86cd510ef2280d70d9d036